src/server/models/Meter.js

```diff
@@ -91,6 +91,13 @@ class Meter {
 		this.id = resp.id;
 	}
 
+	static insertAll(meters, conn = db) {
```

Comment:
You should write a test for this.

Reply:
This method is not used anywhere. I will just remove this.